src/private.js

```diff
@@ -463,7 +457,7 @@ module.exports = (dht) => ({
   },
 
   async _verifyRecordOnlineAsync (record) {
-    return promisify(cb => libp2pRecord.validator.verifyRecord(dht.validators, record, cb))()
+    await promisify(cb => libp2pRecord.validator.verifyRecord(dht.validators, record, cb))()
```

Comment:
I don't know if there are any significant perf impacts, but i find this more readable

when i see `async` i can clearly tell it returns a promise
when i see `await` instead of a `return` i can clearly see it doesn't return any value

Reply:
In IPLD I use `async` to signal it returns a promise, but use in those cases `return` (this is also what our currently force you into).

Reply:
Got it! I typically prefer to have docs for improving readability in those scenarios, but we can iterate on this!